### PR TITLE
Add plugin overview

### PR DESCRIPTION
**Pull request by KnorpelSenf**
As discussed in https://t.me/grammyjs/63737

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
@roj1512 is there anything that's missing here?

**Comment by KnorpelSenf** [comment]
Perhaps we should sort the list in the preferable installation order?

**Comment by roj1512** [comment]
@KnorpelSenf me

**Comment by roj1512** [comment]
I am missing here

**Review by roj1512 — APPROVED** [review]

**Review by roj1512 — APPROVED** [review]
Thanks everyone!

File: site/docs/es/plugins/README.md [hunk 1]
```diff
@@ -21,6 +21,40 @@ Cada sección de la documentación de un plugin oficial tiene el nombre del paqu
...
+| Parsear                            | [`parse-mode`](./parse-mode.md)                       | Simplificar el formato de los mensajes                                      |
+| Limitador de velocidad             | [`ratelimiter`](./ratelimiter.md)                     | Restringe automáticamente a los usuarios que envían spam a tu bot           |
+| Enrutador                          | [`router`](./router.md)                               | Dirija los mensajes a diferentes partes de su código                        |
+| Concurrencia                       | [`runner`](./runner.md)                               | Long polling csimultáneo a escala                                           |
+| Preguntas sin estado               | [`stateless-question`](./stateless-question.md)       | Crear diálogos sin almacenamiento de datos                                  |
+| Control de flujo                   | [`transformer-throttler`](./transformer-throttler.md) | Poner en cola automáticamente las llamadas a la API para evitar las esperas |
+
+También tenemos algunos plugins de terceros.
+Puedes encontrarlos en el menú de navegación en _Plugins_ > _Third-Party_.
```
Comment (CikiMomogi) [inline site/docs/es/plugins/README.md:55]: > _Plugins_ > _Third-Party_

@habemuscode is this navigation menu correct? I think the correct one for ES page is `_Complementos_ > _De Terceros_`.

**Review by CikiMomogi — COMMENTED** [review]
cmiiw

File: site/docs/es/plugins/README.md (hunk 1 above)
Comment (habemuscode) [reply site/docs/es/plugins/README.md:55]: Will be fixed in https://github.com/grammyjs/website/pull/558

Thanks

**Pull request merged**